scripts/pca.py

```diff
@@ -175,7 +177,7 @@ def main(fmat, fxyz, ftags, fcolor, colorscol, prefix, output, peratom, keepraw,
     parser.add_argument('--output', type=str, default='matrix', help='The format for output files ([xyz], [matrix])')
     parser.add_argument('--peratom', type=str2bool, nargs='?', const=True, default=False,
                         help='Do you want to output per atom pca coordinates (True/False)?')
-    parser.add_argument('--keepraw', type=str2bool, nargs='?', const=True, default=False,
+    parser.add_argument('--keepraw', type=str2bool, nargs='?', const=True, default=True,
```

Comment:
Not sure when this was changed. If I accidentally introduced it I can switch it back.

Reply:
Let's make the default false. Otherwise the output file is usually very large

Reply:
Great, I'll switch it back!